src/plone/restapi/services/locking/configure.zcml

```diff
@@ -1,17 +1,46 @@
 <configure
-    xmlns="http://namespaces.zope.org/zope"
-    xmlns:plone="http://namespaces.plone.org/plone"
-    xmlns:zcml="http://namespaces.zope.org/zcml"
-    >
+  xmlns="http://namespaces.zope.org/zope"
+  xmlns:plone="http://namespaces.plone.org/plone"
+  >
+
+  <adapter
+      factory=".get.Locking"
+      name="lock"
+      />
+
+  <plone:service
+      method="GET"
+      factory=".get.Lock"
+      for="zope.interface.Interface"
+      permission="zope2.View"
+      name="@lock"
+      />
 
   <plone:service
       method="POST"
-      factory=".locking.Lock"
-      for="Products.CMFCore.interfaces.IContentish"
```

Comment:
@avoinea why did we move from Contentish to zope Interface here? Also for the other services. Locking does not work on the SiteRoot anyways, right?

Reply:
@tisto True. But in the front-end the Edit form is generic and it sends a `@lock` request which would return 404 if the `@lock` endpoint is not registered on a more generic interface.

Reply:
@avoinea can you elaborate? I don't get it. :)

Reply:
I think we may not need this if we agree on #1201